components/card.css

```diff
@@ -0,0 +1,66 @@
+
+.card-shadow-sm {
+    box-shadow: 0 0 5px -2px;
+}
+
+.card-wrapper {
+    width: 22rem;
+    background: var(--c-white);
+}
+
+.icon-close {
+    color: var(--c-darkgray);
+}
+
+.icon-close:hover {
+    cursor: pointer;
+    color: var(--c-white);
+}
+
+.card-image {
+    width: 100%;
+    object-fit: cover;
+}
+
+.card-3-image {
```

Comment:
Can use better naming. as `card-3` is a bit vague as to what this card is. Rest well-written code. Nice work! 👍

Reply:
ah! i forgot to remove this class as it isn't getting used anymore. thanks for pointing it out.